### PR TITLE
chore: login button position and color on testing environment

### PR DESCRIPTION
**Pull request by EugeneDevv**

### CODE REVIEW

**Comment by EugeneDevv** [comment]
Screenshot_20220319-103629

**Comment by EugeneDevv** [comment]
Screenshot_20220319-104120

File: lib/presentation/onboarding/login/pages/phone_login_page.dart [hunk 1]
```diff
@@ -1,21 +1,306 @@
...
+      );
+
+      await StoreProvider.dispatch<AppState>(
+        context,
+        PhoneLoginAction(
+          context: context,
+        ),
+      );
+    }
+  }
```
Comment (abiudrn) [inline lib/presentation/onboarding/login/pages/phone_login_page.dart:76]: Replace this function by saving the values to state directly in the `onChanged` and then dispatch the second action in the login button directly. Reference the consumer code for more info

**Review by abiudrn — CHANGES_REQUESTED** [review]

File: lib/presentation/onboarding/login/pages/phone_login_page.dart (hunk 1 above)
Comment (EugeneDevv) [reply lib/presentation/onboarding/login/pages/phone_login_page.dart:76]: okay

**Review by abiudrn — APPROVED** [review]

**Review by byronkimani — APPROVED** [review]

**Review by Paulmburu — APPROVED** [review]

**Pull request merged**